README.MD

```diff
@@ -1 +1,163 @@
+1. Install Visual Studio Code from https://code.visualstudio.com/
+
+2. Install Anaconda from https://www.anaconda.com/download
+
+3. Open Visual Stuido Code, go to the extensions tab on the activity bar, and download the the following extensions:
+  * Make sure all of these extensions are by Microsoft:
+    - Python
+    - Jupyter
+    - SQL Server (mssql)
+    - SQL Database Projects
+    - GitHub Pull Requests
+
+5. Congratulations! You are now ready to code!
+
+6. If you need to download data files for your curicullumn lecture, you can find them here: https://drive.google.com/drive/folders/1PIjOqFCnovo8kGbGQDfjFE17BAl_PtqF?usp=drive_link
+
+## Forking the curicullumn
+1. <b>Fork the Repository on GitHub:</b>
+  * Click on the GitHub link shared, which will open the repository in web browser.
+  * Click the "Fork" button at the top right corner of the repository page. 
+
+This will create a personal copy of the repository under your GitHub account.
+
+
+2. <b>Clone the Forked Repository to VSCode:</b>
+  * <b>Open VSCode</b>:
+  * <b>Open the Command Palette</b>:
+    - Press "Ctrl + Shift + P" (or "Cmd + Shift + P" on Mac) to open the Command Palette.
+  * <b>Clone the Repository:</b>
+    - Type "Git: Clone" and select it.
+    - Paste the URL of your forked repository from your GitHub account and choose a local folder where you want to save the repository.
+  * <b>Open the Repository</b>:
+    - Once the repository is cloned, VSCode will prompt to open the repository. Click "Open" to open the project in VSCode.
+
+
+3.<b>Explore and Work on the Materials:</b>
+  * you can now browse through the folders and files in the repository from the VSCode sidebar.
+
+
+4. <b>Open Files and Edit:</b>
+  * They can open any .py or .ipynb files and start working on the exercises, making changes in your local copy.
+
+
+5. <b>Run the Code:</b>
+  * To run Python scripts, you can right-click the file and choose “Run Python File in Terminal” or use the run button at the top right corner.
+  * For Jupyter notebooks (.ipynb files), you can run the cells directly in VSCode if the Jupyter extension is installed.
+
+
+6. <b>Commit Changes Locally:</b>
+  * <b>Make Changes:</b> you can make changes to the files as you work through the exercises.
+  * <b>Stage and Commit Changes:</b> In the Source Control tab, you can stage your changes (click on the +icon next to each file) and commit with a message (e.g., “Completed exercise 1”).
+
+
+7. <b>Push Changes to Their Forked Repository:</b>
+  * <b>Push to GitHub:</b> After committing changes, you can push the changes to your forked repository on GitHub by clicking the "Sync Changes" button in the Source Control tab.
+  * This allows you to save your progress online without affecting the original repository.
+
+8. If you need a more visual guide to understand how to fork a repository, you can find it here: https://scribehow.com/shared/Forking_the_DACDSA_Curriculum_Repository__uJ4bd99_TTWagei2bQgC1Q
```

Comment:
@ Vettel99 I suggest you write the tag at the top instead of the bottom from a reader's standpoint, we read from top to bottom